anitya/db/models.py

```diff
@@ -656,7 +658,9 @@ class ProjectVersion(Base):
     )
     version = sa.Column(sa.String(50), primary_key=True)
 
-    project = sa.orm.relation('Project', backref='versions_obj')
+    project = sa.orm.relation(
```

Comment:
Maybe you could use the relationship API here like in [here](https://github.com/release-monitoring/anitya/pull/608/files#diff-426a0c336a142390e6230960f40efb78R352)

Reply:
`relation` is the synonym to `relationship`. See here http://docs.sqlalchemy.org/en/latest/orm/relationship_api.html

Reply:
So why not keep consistency and use the same name either `relation` or `relationship` I think it would make the code easier to read.

Reply:
As you can see I used the same method that was already there, but you are right I didn't noticed this.